brambling/templates/brambling/forms/event.html

```diff
@@ -8,19 +8,21 @@ <h4 class="panel-title">Information</h4>
 	<div class="panel-body">
 		{% formrow form.name %}
 		{% if form.slug %}
-			{% formrow form.slug %}
+			{% formrow form.slug with prepend="https://dancerfly.com/"|add:organization.slug|add:"/" label="URL" %}
```

Comment:
/me grumbles about hard-coding the domain and protocol into the template, then realizes it will probably never matter